### PR TITLE
Add HUI-audio-corpus-german recipe for ESPnet2-TTS

### PR DESCRIPTION
**Pull request by kan-bayashi**
This PR adds a new recipe of HUI audio corpus german.
https://opendata.iisys.de/datasets.html#hui-audio-corpus-german

### CODE REVIEW

File: egs2/TEMPLATE/asr1/db.sh [hunk 1]
```diff
@@ -27,6 +27,7 @@ CSJVER=dvd  ## Set your CSJ format (dvd or usb).
 CSMSC=downloads
 HKUST1=
 HKUST2=
+HUI_ACG=downloads
```
Comment (ftshijt) [inline egs2/TEMPLATE/asr1/db.sh:30]: Could you also reserve a blank entry in the JHU environment?

**Review by ftshijt — COMMENTED** [review]
I think it's in a good shape. Just please remember to update the egs/README.md with the dataset description

File: egs2/TEMPLATE/asr1/db.sh (hunk 1 above)
Comment (kan-bayashi) [reply egs2/TEMPLATE/asr1/db.sh:30]: Oh sorry. I forgot to add it.

**Comment by ftshijt** [comment]
LGTM! Will you upload the pre-trained model to this recipe as well?

**Comment by kan-bayashi** [comment]
Training is on-going. Once finished, I will upload :)

**Comment by kan-bayashi** [comment]
I will upload results in another PR. Let me merge this PR.

**Pull request merged**